handlers/data.go

```diff
@@ -1,6 +1,6 @@
 package handlers
 
-import "git.jojii.de/jojii/zimserver/zim"
+import "git.jojii.de/jojii/ZimWiki/zim"
```

Comment:
I'm wondering why this still works... This url isn't valid anymore... maybe we should replace it with the github url to avoid potential compilation errors in future?

Reply:
Indeed, I think we can do that. :)
I thought there was a particular reason for that, so I didn't touch it.
In fact, since the files are directly in the local folder, it does not need to search for them from remote repository.
This may explains why the compilation kept working...